tests/test_dictionaries.py

```diff
@@ -0,0 +1,15 @@
+import string
+
+
+def get_dictionary(k, v):
```

Comment:
These unit tests are failing with pytest:

```
=========================================================== test session starts ============================================================
platform linux -- Python 3.6.7, pytest-5.0.1, py-1.8.0, pluggy-0.12.0
rootdir: /home/lao/dev/mozilla/bugzilla-dashboard-flask
collected 2 items                                                                                                                          

tests/test_dictionaries.py FF                                                                                                        [100%]

================================================================= FAILURES =================================================================
___________________________________________________ test_big_dictionary_different_value ____________________________________________________

    def test_big_dictionary_different_value():
>       assert get_dictionary('a', 'def') == get_dictionary('a', 'abc')
E       AssertionError: assert {'\t': 9, '\n...x0c': 12, ...} == {'\t': 9, '\n'...x0c': 12, ...}
E         Omitting 99 identical items, use -vv to show
E         Differing items:
E         {'a': 'def'} != {'a': 'abc'}
E         Use -v to get the full diff

tests/test_dictionaries.py:11: AssertionError
____________________________________________________ test_big_dictionary_differnt_keys _____________________________________________________

    def test_big_dictionary_differnt_keys():
>       assert get_dictionary('abc', 1) == get_dictionary('def', 2)
E       AssertionError: assert {'\t': 9, '\n...x0c': 12, ...} == {'\t': 9, '\n'...x0c': 12, ...}
E         Omitting 100 identical items, use -vv to show
E         Left contains 1 more item:
E         {'abc': 1}
E         Right contains 1 more item:
E         {'def': 2}
E         Use -v to get the full diff

tests/test_dictionaries.py:15: AssertionError
========================================================= 2 failed in 0.04 seconds ========================================================
```

How could they be valid as you change the dictionnaries with different value.

For a dummy test, simply do:

```
def test_dummy():
  assert True
```